test/e2e/parallel/e2e_test.go

```diff
@@ -284,6 +285,20 @@ func TestE2EFlow(t *testing.T) {
 		require.NoError(t, err)
 		require.NotEmpty(t, cm)
 
+		// ensure that the finalizer is removed in all cases, so the cleanup logic
+		// is not blocked and can delete the Space
+		var removeOnce sync.Once
```

Comment:
same as in the other PR - it should be safe but maybe we could move this definition up one level.

Reply:
and the same answer - it should stay in the save sub-test where the ConfigMap is provisoned & modified